src/transaction/algo.ts

```diff
@@ -0,0 +1,54 @@
+const algosdk = require('algosdk');
+const base32 = require('base32.js');
+
+export const getClient = (testnet: boolean) => {
+    if (testnet) {
+        const baseServer = 'https://testnet-algorand.api.purestake.io/ps2'
```

Comment:
make this as optional parameter - provider?: string, which defaults to TATUM_API_URL/v3/algorand/node